### PR TITLE
chore: update api docstring

### PR DESCRIPTION
**Pull request by williamstar**
#23

### CODE REVIEW

**Review by NOBLES5E — APPROVED** [review]

File: persia/ctx.py [hunk 1]
```diff
@@ -240,21 +231,22 @@ class TrainCtx(BaseCtx):
 
     def __init__(
         self,
-        grad_scaler: torch.cuda.amp.GradScaler = None,
+        grad_scaler: Optional[torch.cuda.amp.GradScaler] = None,
```
Comment (snowpeakz) [inline persia/ctx.py:234]: Better to move all optional args behind to the required?

File: persia/data.py [hunk 2]
```diff
@@ -77,13 +78,14 @@ class PersiaChannel(IterableChannelBase):
...
-        replica_info (PyPersiaReplicaInfo): replica info of current process to enable the data reorder ability
+        replica_info (PyPersiaReplicaInfo, optional): replica info of current process to enable the data reorder ability
+        async_iterator (bool, optional): launch the thread to generate the data asynchronous
     """
 
     def __init__(
         self,
         buffer_size: int,
-        replica_info: PyPersiaReplicaInfo = None,
+        replica_info: Optional[PyPersiaReplicaInfo] = None,
```
Comment (snowpeakz) [inline persia/data.py:88]: Better to move all optional args behind to the required?

File: persia/ctx.py (hunk 1 above)
Comment (williamstar) [reply persia/ctx.py:234]: If the `CtxStatus` is `CtxStatus.EVAL` or `CtxStatus.INFERENCE`, there may not required the grad_scalar

File: persia/data.py (hunk 2 above)
Comment (williamstar) [reply persia/data.py:88]: `Optional` means that it can set to `None` as default value to pass the pytype test

**Review by snowpeakz — APPROVED** [review]

**Pull request merged**